src/api/data-migration/version-1-to-version-2/get-all-buyers.ts

```diff
@@ -0,0 +1,23 @@
+import { Connection } from 'mysql2/promise';
+import executeSqlQuery from './execute-sql-query';
+import { ApplicationBuyerMvp } from '../../types';
+
+/**
+ * getAllBuyers
+ * Get all buyers in the "Buyer" table.
+ * @param {Connection} connection: SQL database connection
+ * @returns {Promise<Array<object>>} executeSqlQuery response
+ */
+const getAllBuyers = async (connection: Connection) => {
+  const loggingMessage = 'Getting all buyers';
+
+  const query = 'SELECT * FROM Buyer';
```

Comment:
could we not do const buyers = (await context.db.Buyer.findMany()) as Array<object>;?

Reply:
No, we're doing this in several places for any data models that have been split up as part of no PDF - e.g with `AccountStatus`.

In MVP, Buyer is just one table. In no-pdf, fields have moved out of the Buyer table - there are 3x new tables with some fields in the original Buyer table, moved into the new tables.

During migration, for these instances (of data moving around), we need to get the current state of the database.

This has to be done with a raw SQL query - Keystone `findMany()` (and other context queries), will not return all the data, since the keystone schema has changed (where the fields have moved).

So after obtaining the raw database data for e.g in this instance, `Buyer`, we use the data to then create the new relationships and tables etc. Afterwards, the fields in the original database are deleted.